index.html

```diff
@@ -1018,6 +1030,13 @@ <h2>
             </li>
           </ol>
         </dd>
+        <dt>
+          <dfn>type</dfn> attribute
+        </dt>
+        <dd>
+          An enumeration, {{GamepadButtonType}}, that indicates which button types the controller
+          is being held in.
+        </dd>
```

Comment:
Looks like we're still missing getter definitions, no?

Reply:
Yeah, though it's implied... but yeah, if the others have the getter steps (what it was initialized to) then so should this.